Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get update \
  && chmod 777 /cache /config /media
 COPY --from=ffmpeg / /
 COPY --from=builder /jellyfin /jellyfin
-COPY --from=web-builder /dist /jellyfin/jellyfin-web/src
+COPY --from=web-builder /dist /jellyfin/jellyfin-web
```

Comment:
This will therefore copy the entire project right?

Reply:
Well no, it's just the target folder nolonger needs src. In the CI I move the contents of the dist folder directly to the jellyfin-web folder just like here. I also adjusted the default web dir in the server C# code.
Realistically the csproj really shouldn't need to include these files anyway, but it makes for easier publishing so... Full separation can be achieved fairly easily at least on the build side. But that is for another day.